lib/index.js

```diff
@@ -37,7 +37,15 @@ module.exports = (options = {}) => {
   })
 
   // Log all webhook errors
-  webhook.on('error', logger.error.bind(logger))
+  webhook.on('error', (...args) => {
```

Comment:
Is this ever more than a single argument?

Reply:
I'm not sure 🙁 I felt it was better to be safe than sorry; worst case scenario an important error leaves out the second argument and confusion happens. I also really don't like this, so I'm happy to change it to only accept one arg.